prowler/providers/aws/services/inspector2/inspector2_service.py

```diff
@@ -0,0 +1,115 @@
+import threading
+
+from botocore.client import ClientError
+from pydantic import BaseModel
+
+from prowler.lib.logger import logger
+from prowler.lib.scan_filters.scan_filters import is_resource_filtered
+from prowler.providers.aws.aws_provider import generate_regional_clients
+
+
+################################ Inspector2
+class Inspector2:
+    def __init__(self, audit_info):
+        self.service = "inspector2"
+        self.session = audit_info.audit_session
+        self.audited_account = audit_info.audited_account
+        self.audit_resources = audit_info.audit_resources
+        self.audited_partition = audit_info.audited_partition
+        self.regional_clients = generate_regional_clients(self.service, audit_info)
+        # If the region is not set in the audit profile,
+        # we pick the first region from the regional clients list
+        self.region = (
+            audit_info.profile_region
+            if audit_info.profile_region
+            else list(self.regional_clients.keys())[0]
+        )
+        self.inspectors = []
+        self.__get_configuration__()
+        self.__list_findings__()
+
+    def __get_session__(self):
+        return self.session
+
+    def __threading_call__(self, call):
+        threads = []
+        for regional_client in self.regional_clients.values():
+            threads.append(threading.Thread(target=call, args=(regional_client,)))
+        for t in threads:
+            t.start()
+        for t in threads:
+            t.join()
+
+    def __get_configuration__(self):
+        # We use this function to check if inspector2 is enabled
+        logger.info("Inspector2 - get configuration...")
+        try:
+            for regional_client in self.regional_clients.values():
+                try:
+                    regional_client.get_configuration()
+                    self.inspectors.append(
+                        Inspector(
+                            id="Inspector2",
+                            region=regional_client.region,
+                        )
+                    )
+                except ClientError as error:
+                    if error.response["Error"]["Code"] == "ResourceNotFoundException":
```

Comment:
Is this a global service? If not we need to create a dummy object per instance to set the status of the inspector service and then check it. What do you think?